src/serialization/QuerySerializer.ts

```diff
@@ -0,0 +1,10 @@
+import QueryRepresentation from '@/sparql/QueryRepresentation';
+
+export default class QuerySerializer {
+	public serialize( query: QueryRepresentation ): object {
```

Comment:
That serializer should probably return a string, not an object.

Reply:
Adressed